client/services/code_evaluators/PythonPrereqCheckService.js

```diff
@@ -213,11 +236,18 @@ tie.factory('PythonPrereqCheckService', [
             PREREQ_CHECK_TYPE_BAD_IMPORT, unsupportedImports, null);
         }
 
+        var wrongLangName = detectAndGetWrongLanguageType(code);
```

Comment:
Can we rename this to wrongLangError or wrongLangErrorName perhaps? I feel like wrongLangName sounds like the name of the language.